MC/ReconstructionConfig.C

```diff
@@ -203,6 +205,16 @@ void ReconstructionConfig(AliReconstruction &rec, int tag_tmp)
     rec.SetRun1PIDforTracking(kTRUE);
     return;
 
+  case kReconstructionForNuclei:
+    ReconstructionDefault(rec);
+    const char* conf = gSystem->Getenv("CONFIG_HE3_PION_THRESHOLD");
+    int threshold = 130;
+    if (conf && !conf[0]) {
```

Comment:
Hi Max, why did you put the condition !conf[0] ?

Reply:
That is the old piece of code, I now replaced it with the one that uses only atoi, thanks for noticing!